app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/api-integration/components/ApiKeyButton.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
```

Comment:
Button existed before but copied over to new locations + modified to extract the key id Tool tip (now placed above the buttons).